language/transaction-builder/src/lib.rs

```diff
@@ -17,22 +17,27 @@ use std::{collections::HashSet, iter::FromIterator};
 use stdlib::{
     stdlib_modules,
     transaction_scripts::{
-        BLOCK_PROLOGUE_TXN_BODY, CREATE_ACCOUNT_TXN_BODY, MINT_TXN_BODY,
+        ADD_VALIDATOR_TXN_BODY, BLOCK_PROLOGUE_TXN_BODY, CREATE_ACCOUNT_TXN_BODY, MINT_TXN_BODY,
         PEER_TO_PEER_TRANSFER_TXN_BODY, PEER_TO_PEER_TRANSFER_WITH_METADATA_TXN_BODY,
-        ROTATE_AUTHENTICATION_KEY_TXN_BODY, ROTATE_CONSENSUS_PUBKEY_TXN_BODY,
+        REGISTER_VALIDATOR_TXN_BODY, REMOVE_VALIDATOR_TXN_BODY, ROTATE_AUTHENTICATION_KEY_TXN_BODY,
+        ROTATE_CONSENSUS_PUBKEY_TXN_BODY,
     },
 };
 #[cfg(any(test, feature = "fuzzing"))]
 use vm::file_format::Bytecode;
 
 lazy_static! {
+    pub static ref ADD_VALIDATOR_TXN: Vec<u8> = { compile_script(&ADD_VALIDATOR_TXN_BODY) };
```

Comment:
different patterns here too :) I don't really get how those layers work together so just delegate to you

Reply:
Me neither, I think we can simplify this a lot...